Library/Formula/qxt.rb

```diff
@@ -0,0 +1,239 @@
+class Qxt < Formula
+  desc "LibQxt is an extension library for Qt"
+  homepage "https://bitbucket.org/libqxt/libqxt/wiki/Home"
+
+  stable do
+    url "https://bitbucket.org/libqxt/libqxt/get/v0.6.2.tar.gz"
+    sha256 "582426d6c81958dd1ac5ca4a9d49807e0a2204b7343e7d49c6613bcb148b5cb8"
+    patch :DATA
+  end
+
+  head do
+    url "https://bitbucket.org/libqxt/libqxt.git"
+
+    patch :p1, <<EOS
+--- a/src/widgets/mac/qxtwindowsystem_mac.cpp
++++ b/src/widgets/mac/qxtwindowsystem_mac.cpp
+@@ -89,11 +89,7 @@ QString QxtWindowSystem::windowTitle(WId window)
+     // most of CoreGraphics private definitions ask for CGSValue as key but since
+     // converting strings to/from CGSValue was dropped in 10.5, I use CFString, which
+     // apparently also works.
+-    err = CGSGetWindowProperty(connection, window, (CGSValue)CFSTR("kCGSWindowTitle"), &windowTitle);
+-    if (err != noErr) return QString();
+-
+-    // this is UTF8 encoded
+-    return QCFString::toQString((CFStringRef)windowTitle);
++    return QString();
+ }
+EOS
+  end
+
+  depends_on "qt"
+  depends_on "berkeley-db" => :recommended
+
+  def install
+    args = ["-prefix", prefix,
+            "-libdir", lib,
+            "-release"]
+
+    if build.without? "berkeley-db"
+      args << "-no-db"
+    end
+
+    system "./configure", *args
+    system "make", "install"
+
+    if build.head?
+      opoo "Note: QxtWindowSystem::windowTitle is currently unsupported and will return QString()"
```

Comment:
We prefer not to print anything during the build; we use `def caveats` for Homebrew-specific messages.